src/smdba/postgresqlgate.py

```diff
@@ -941,7 +941,7 @@ def do_system_check(self, *args, **params):
             changed = True
 
         # max_locks_per_transaction
-        if not conf.get('max_locks_per_transaction'):
+        if not conf.get('max_locks_per_transaction') or conf.get('max_locks_per_transaction') < '100':
```

Comment:
@dmacvicar Because it will crash, if there is no value. :wink: Correct would be:
```python
max(conf.get('max_locks_per_transaction', 100), 100)
```